Bug | PORT-15315 Kubernetes integration is not mandatory in the terraform management guide (#2498)
port-labs/port-docs: docs/guides/all/import-and-manage-integration.md

# Manage integration mapping using Terraform

This guide demonstrates how to use the Import State feature of Terraform to manage your Port integration's mapping with Terraform.

## Prerequisites

> This guide assumes:
> - Complete the [onboarding process](/getting-started/overview).
> - You have the specific [integration](/integrations-index) you want to manage already installed in Port. 

Reviewer: ```suggestion

- Complete the [onboarding process](/getting-started/overview).
- Install the [integration](/integrations-index) whose mapping you want to manage. 
```
